checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInferenceLogic.java

```diff
@@ -43,6 +76,15 @@ public class MustCallInferenceLogic {
   /** The control flow graph. */
   private final ControlFlowGraph cfg;
 
+  /** TODO */
+  private final MustCallConsistencyAnalyzer mcca;
+
+  /** The MethodTree of the cfg. */
+  private MethodTree enMethodTree;
```

Comment:
What is "en" a mnemonic for?

Reply:
It is for representing the enclosing method tree. It is used for analyzing the enclosed statements within the method body.

Reply:
I agree with Mike that the "en" mnemonic is confusing. I recommend just changing the name of this field to `enclosingMethodTree`

Reply:
(and the similar change `enMethodElt` -> `enclosingMethodElt`)